user.go

```diff
@@ -541,6 +541,19 @@ func (user *User) unlockedConnect() {
 			})
 		}
 		go user.sendMarkdownBridgeAlert(context.TODO(), "Failed to connect to %s: %v", user.bridge.ProtocolName, err)
+		return
+	}
+
+	refreshInterval := time.Duration(user.bridge.Config.Meta.ForceRefreshIntervalSeconds) * time.Second
+	if refreshInterval > 0 {
+		user.log.Debug().Msgf("Connection will be refreshed at %s", time.Now().Add(refreshInterval).Format(time.RFC3339))
```

Comment:
Msgf is not allowed (that's why the CI is angry), installing [pre-commit](https://pre-commit.com/) is also recommended to catch linting issues locally

Reply:
Just noticed that, fixed it already :)